edx_name_affirmation/tasks.py

```diff
@@ -96,6 +96,79 @@ def idv_update_verified_name(self, attempt_id, user_id, status, photo_id_name, f
         )
 
 
+@shared_task(
+    bind=True, autoretry_for=(Exception,), default_retry_delay=DEFAULT_RETRY_SECONDS, max_retries=MAX_RETRIES,
+)
+@set_code_owner_attribute
+def idv_update_verified_name_task(self, attempt_id, user_id, name_affirmation_status, photo_id_name, full_name):
```

Comment:
Added new (but still unused) IDV task. This is to avoid any status errors that would come as a result of modifying the original task. Errors in that state would occur because we now want to pass the name affirmation status to the tasks, as opposed to the IDV or proctoring status. By creating a new task, we can allow any old celery tasks to still work as expected with the IDV status, but then cleanly transition to this new task when we want to switch.